status/backend.json

```diff
@@ -61,6 +61,13 @@
         "android": "2.22.0.3"
       }
     },
+    "idPay": {
+      "enabled": false,
+      "min_app_version": {
+        "ios": "2.22.0.3",
```

Comment:
To allow all the version can we set `*`?

Reply:
`VersionPerPlatform` uses the pattern `^\\d+.\\d+.\\d+(.\\d+)?$` to validate versions, so I suppose we should set `"min_app_version": "0.0.0.0"` to allow all versions